cdap-api/src/main/java/co/cask/cdap/api/dataset/table/TableProperties.java

```diff
@@ -233,6 +236,28 @@ public static String getRowFieldName(Map<String, String> props) {
     return props.get(Table.PROPERTY_SCHEMA_ROW_FIELD);
   }
 
+
+  /**
+   * @return the table permissions as a map from user name to a permission string
+   */
+  @Beta
+  @Nullable
+  public static Map<String, String> getTablePermissions(Map<String, String> properties) {
+    String propertyValue = properties.get(PROPERTY_TABLE_PERMISSIONS);
+    if (propertyValue == null) {
+      return null;
```

Comment:
The caller may have a default. If it is null, he would use the default. If it is empty, it would override his default.